src/pivot-table/hooks/use-items-rendered-handler/utils/fetch-pages.ts

```diff
@@ -60,6 +60,14 @@ export const fetchPages = async (
     );
   }
 
+  // A chart have been re-size which have triggered a onScroll event and reset both scroll directions
+  if (
+    verticalScrollDirection.current === ScrollDirection.None &&
+    horizontalScrollDirection.current === ScrollDirection.None
```

Comment:
Is this only true when the chart has been resized. Do we always fetch pages when one of the scroll directions are not `None`

Reply:
It will be `None` for both in the default state. So when the chart is first rendered. But should end with all data already being fetched. So no additonal fetch there.